### PR TITLE
Bump default --kubernetes-version to v1.25.3

### PR DESCRIPTION
**Pull request by mboersma**
**Description**

Updates the default value for `az capi create --kubernetes-version` to v1.25.3.

**History Notes**

```
Default `az capi create --kubernetes-version` to v1.25.3
```

---

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).
- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).
- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by mboersma** [comment]
/assign @marosset

I think this is a safe change, but since `az capi` is used in Windows pipeline tests, I just want to ensure this won't cause problems for your usage. It's just changing the default from v1.22.8 to v1.25.3.

**Review by willie-yao — APPROVED** [review]
/lgtm

**Comment by jsturtevant** [comment]
> I think this is a safe change, but since `az capi` is used in Windows pipeline tests, I just want to ensure this won't cause problems for your usage. It's just changing the default from v1.22.8 to v1.25.3.

we overload this value with the nightly build version anyways so should be fine

**Comment by marosset** [comment]
> this is a safe change, but since `az capi` is us

+1 this should be fine.

File: src/capi/setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 # Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.0.6'
+VERSION = '0.0.7'
```
Comment (marosset) [inline src/capi/setup.py:20]: Should we make a v0.1.0 release?

We are starting to take more dependencies on this project for SIG-Windows testing in Kubernetes

Comment (mboersma) [reply src/capi/setup.py:20]: Sure, that would make it possible to do start doing proper patch releases. Maybe we can do the next release as v0.0.7 and then the next is v0.1.0?

Comment (mboersma) [reply src/capi/setup.py:20]: Scratch that, I'll update this so the next release is v0.1.0.

**Review by marosset — APPROVED** [review]
/lgtm

**Pull request merged**